roles/ewatercycle/templates/environment.yml.j2

```diff
@@ -21,6 +21,8 @@ dependencies:
       # Conda version of papermill is not py3.12 compat, so use pip version
       - papermill==2.6.0
       - rich==13.9.4
+      - folium==0.20.0
+      - geopandas==1.1.1
```

Comment:
I think it might be better to put these in the conda dependencies, not pip. That would reduce the chance of environment conflicts.

Reply:
no just slightly lower in this same file. Below the `  # Jupyter` comment.